src/pages/group/index.tsx

```diff
@@ -54,8 +69,13 @@ const GroupPage = () => {
           handleChange={handleChange}
           handleSumbit={handleSumbit}
         />
-        {isSuccess && <GroupList bookGroups={data.bookGroups} />}
+        {isSuccess &&
+          data.pages.map((groups, idx) => {
+            return <GroupList key={idx} bookGroups={groups.bookGroups} />;
+          })}
       </Box>
+      <Box ref={ref} />
+      {isFetchingNextPage ? <Box>Loading...</Box> : null}
```

Comment:
```suggestion
      {isFetchingNextPage ? <Skeleton w="100%" height="28rem" /> : null}
```

이렇게 작성하면 스켈레톤 코드가 적용되는 것 같아요! 아마 스켈레톤 코드의 영역을 지정해주지 않아서 그랬던 것 같습니다~

Reply:
네 감사합니다! 적용해보고 수정하겠습니다~👍

Reply:
스켈레톤으로 변경 적용했습니다~ 😁 리뷰 부탁드려용!